recipes/cad1/task1/baseline/evaluate.py

```diff
@@ -199,20 +200,19 @@ def _evaluate_song_listener(
             frequencies=frequencies,
         )
 
-        #  audiogram, audiogram_frequencies, fs_signal
-        per_instrument_score[f"left_{instrument}"] = compute_haaqi_rms(
+        per_instrument_score[f"left_{instrument}"] = compute_haaqi(
             left_enhanced_signal,
             reference_signal[:, 0],
             audiogram_left,
             config.sample_rate,
-            silence_length=2.0,
+            65 - 10 * np.log10(compute_rms(reference_signal[:, 0])),
```

Comment:
This looks wrong to me. If you are working with an rms amplitude value then the dB level needs 20* rather than 10*.